factory-girl/src/main/scala/skinny/test/FactoryGirl.scala

```diff
@@ -51,8 +53,22 @@ case class FactoryGirl[Id, Entity](mapper: CRUDFeatureWithId[Id, Entity], name:
    * @return attributes in conf file
    */
   def loadedAttributes(): Map[SQLSyntax, Any] = {
-    // TODO directory scan and work with factories/*.conf
-    val config = ConfigFactory.parseResources(getClass.getClassLoader, "factories.conf").resolve().getConfig(factoryName.name)
+    val rootResolve = ConfigFactory.parseResources(getClass.getClassLoader, "factories.conf").resolve()
```

Comment:
How about `rootResolvedConfig`? The `resolve` method returns a replacement config with all substitutions. https://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#resolve--